CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- ERC20Permit component and preset (#1140)
```

Comment:
We need to update this. Also adding subentries will be helpful to users, like the embeddable implementations added and the utilities.

Reply:
Updated